calcom/cal.com
# chore: allow disabling source map generation with env var

## What does this PR do?

Adds env var that allows to disable server source maps generation and significantly reduce memory usage.

## Requirement/Documentation

[Document](https://www.notion.so/intuita/Build-crash-issue-fcf221fe0abd430795b667cda5ad7f32?pvs=4#b7d8a19b8df34aaa920b9d0cf35fba55)
See sentry plugin section. 

## Type of change




## How should this be tested?

set SENTRY_DISABLE_SERVER_WEBPACK_PLUGIN to 1 and build the project. Source maps should not be generated. Build should not crash. 
## Mandatory Tasks
